assets/src/data/content/model/elements/types.ts

```diff
@@ -8,10 +8,13 @@ interface SlateElement<Children extends Descendant[]> extends BaseElement, Ident
 
 export type ModelElement = TopLevel | Block | Inline;
 
+// A list of all our element types, including those that can't be "bare" inside a children array.
+export type AllModelElements = ModelElement | SubElements;
```

Comment:
I did some work getting

priv/schemas/v0-1-0/content-element.schema.json
assets/src/data/content/model/schema.ts
assets/src/data/content/model/elements/types.ts

all in better sync/alignment with each other. This was part of it.  The goal of this change was to separate out the type that can be in our content model with all of the types of all the renderable elements.